server/src/main/java/org/elasticsearch/index/store/LuceneFilesExtensions.java

```diff
@@ -165,6 +165,11 @@ public static LuceneFilesExtensions fromExtension(String ext) {
         return null;
     }
 
+    @Nullable
+    public static boolean isLuceneExtension(String ext) {
+        return extensions.containsKey(ext);
```

Comment:
nit: this is not nullable

Reply:
Oh yes. I see what you mean. 

We should `@Nullable` the method parameter ++ 

Being on the method it indicates the method can return null.